toolkits/search/arcade_search/tools/google_finance.py

```diff
@@ -0,0 +1,81 @@
+from typing import Annotated, Any
+
+from arcade.sdk import ToolContext, tool
+
+from arcade_search.enums import GoogleFinanceWindow
+from arcade_search.utils import call_serpapi, prepare_params
+
+
+@tool(requires_secrets=["SERP_API_KEY"])
+async def get_stock_summary(
+    context: ToolContext,
+    ticker_symbol: Annotated[str, "The stock ticker to get summary for (e.g., GOOG)"],
+    exchange_identifier: Annotated[
+        str,
+        "The exchange identifier. This part indicates the market where the "
+        "stock is traded (e.g., NASDAQ)",
+    ],
+) -> Annotated[dict[str, Any], "Summary of the stock's recent performance"]:
+    """
+    Retrieve the summary information for a given stock ticker using the Google Finance API.
```

Comment:
Get what information? be more specific so it picks from the context better. 

ex. "Retrieve the Profit to Earnings (P/E) ratio, market capitalization, and Ebitda (Sam: I'm spelling that wrong)."

Reply:
improved function name and docstring